editor/plugins/script_text_editor.cpp

```diff
@@ -1745,6 +1745,45 @@ static String _get_dropped_resource_line(const Ref<Resource> &p_resource, bool p
 	return vformat("const %s = preload(%s)", variable_name, _quote_drop_data(path));
 }
 
+static String _get_path_to_node(const Node *p_from, const Node *p_to) {
+	bool is_unique = false;
+	String path;
+	if (p_to->is_unique_name_in_owner()) {
+		path = p_to->get_name();
+		is_unique = true;
+	} else {
+		path = p_from->get_path_to(p_to);
+	}
+
+	for (const String &segment : path.split("/")) {
+		if (!segment.is_valid_identifier()) {
+			path = _quote_drop_data(path);
+			break;
+		}
+	}
+	return (is_unique ? "%" : "$") + path;
+}
+
+static bool _is_dict_or_array(const Variant &p_value) {
+	bool result = false;
+	switch (p_value.get_type()) {
+		case Variant::Type::DICTIONARY:
+		case Variant::Type::ARRAY:
+		case Variant::Type::PACKED_BYTE_ARRAY:
+		case Variant::Type::PACKED_COLOR_ARRAY:
+		case Variant::Type::PACKED_INT32_ARRAY:
+		case Variant::Type::PACKED_INT64_ARRAY:
+		case Variant::Type::PACKED_FLOAT32_ARRAY:
+		case Variant::Type::PACKED_FLOAT64_ARRAY:
+		case Variant::Type::PACKED_STRING_ARRAY:
+		case Variant::Type::PACKED_VECTOR2_ARRAY:
+		case Variant::Type::PACKED_VECTOR3_ARRAY: {
+			result = true;
+		} break;
```

Comment:
```suggestion
		case Variant::Type::PACKED_VECTOR3_ARRAY:
			result = true;
			break;
		default:
			break;
```

_28 enumeration values not handled in switch: `NIL`, `BOOL`, `INT`..._

The `default` statement should take care of that (at least I hope so).